common/lib/xmodule/xmodule/html_module.py

```diff
@@ -79,6 +79,13 @@ def student_view(self, _context):
         """
         return Fragment(self.get_html())
 
+    @XBlock.supports("multi_device")
+    def anonymous_view(self, _context):
```

Comment:
There is an underscore in `student_view(self, _context)` because the parameter is not being used but since you are using it in the method, make it `context`.

Reply:
Oh yeah, was just blindly following the api 👍